src/main/java/com/team2502/robot2017/subsystem/DriveTrainSubsystem.java

```diff
@@ -86,61 +86,39 @@ else if(xLevel < 0)
      * @param isLeftSide Whether or not it is the left joystick/side
      * @return the speed of the robot
      */
-    private double getSpeed(boolean isLeftSide)
+    private Pair<Double, Double> getSpeed()
     {
         double joystickLevel = 0.0D;
         // Get the base speed of the robot
-        joystickLevel = isLeftSide ? -OI.JOYSTICK_DRIVE_LEFT.getY() : -OI.JOYSTICK_DRIVE_RIGHT.getY();
+        joystickLevel = -OI.JOYSTICK_DRIVE_LEFT.getY();
 
         // Only increase the speed by a small amount
-        if(isLeftSide)
-        {
-            double diff = joystickLevel - lastLeft;
-            if(diff > 0.1D) { joystickLevel = lastLeft + 0.1D; }
-            else if(diff < 0.1D) { joystickLevel = lastLeft - 0.1D; }
-            lastLeft = joystickLevel;
-        }
-        else
-        {
-            double diff = joystickLevel - lastRight;
-            if(diff > 0.1D) { joystickLevel = lastRight + 0.1D; }
-            else if(diff < 0.1D) { joystickLevel = lastRight - 0.1D; }
-            lastRight = joystickLevel;
-        }
+        double diff = joystickLevel - lastLeft;
+        if(diff > 0.1D) { joystickLevel = lastLeft + 0.1D; }
```

Comment:
style comment, so take with take with grains of salt: 
One of the reasons for a style like:

    if(diff > 0.1D) {
             joystickLevel = lastLeft + 0.1D;
     } else if(diff < 0.1D) {
            joystickLevel = lastLeft - 0.1D;
    }

is that when you are look at reviews, it's easier to see _what_ changed. It's kinda no biggie here, but IRL I'm actually taking a break from reviewing a chain of 2000 line diffs.